apps/block_scout_web/lib/block_scout_web/templates/block/_tile.html.eex

```diff
@@ -20,7 +20,7 @@
             #<%= @block %>
           <% end %>
       <% end %>
-      <span class="tile-status-label font-weight-400" data-test="transaction_type">Batch</span>
+      <span class="tile-status-label font-weight-400" data-test="transaction_type">Block</span>
```

Comment:
Don't know who changed that... It was a string literal already. I'm changing it to the string template.

Reply:
> Don't know who changed that...
Me 😆

In this case, the string "Block" was extracted from the property `block_type` so I changed it to the literal `Batch`. Now that we're reverting these changes, we better go back to the original code to avoid future conflicts.